be1-go/hub/standard_hub/mod.go

```diff
@@ -85,6 +111,10 @@ func NewHub(public kyber.Point, log zerolog.Logger, laoFac channel.LaoFactory) (
 		workers:         semaphore.NewWeighted(numWorkers),
 		log:             log,
 		laoFac:          laoFac,
+		serverSockets:   channel.NewSockets(),
+		hubInbox:        *inbox.NewInbox(rootChannel),
+		rootInbox:       *inbox.NewInbox(rootChannel),
+		queries:         queries{state: make(map[int]*bool), queries: make(map[int]method.Catchup), nextID: 0},
```

Comment:
declare and use a `newQueries` constructor instead. This constructor must be right above the struct declaration.